gst/droidcodec/gstdroidvdec.c

```diff
@@ -674,11 +709,13 @@ gst_droidvdec_configure_state (GstVideoDecoder * decoder, guint width,
   droid_media_codec_get_output_info (dec->codec, &md, &rect);
 
   GST_INFO_OBJECT (dec,
-      "codec reported state: width: %d, height: %d, crop: %d,%d %d,%d",
-      md.width, md.height, rect.left, rect.top, rect.right, rect.bottom);
+      "codec reported state: colour: %d, width: %d, height: %d, crop: %d,%d %d,%d",
+      md.hal_format, md.width, md.height, rect.left, rect.top, rect.right,
+      rect.bottom);
 
   dec->codec_reported_height = md.height;
   dec->codec_reported_width = md.width;
+  dec->hal_format = md.hal_format;
```

Comment:
Is the hal_format value new? I swear I couldn't find any such thing when I added the inline conversion, hence the assumption of NV12.

Reply:
Okay, I've read the bug now and it is new.